learning_journal/views.py

```diff
@@ -35,9 +36,16 @@ def create(request):
         return HTTPFound(location=request.route_url('home'))
     return {'form': form, 'action': request.matchdict.get('action')}
 
-@view_config(route_name='action', match_param='action=edit', renderer='string')
+@view_config(route_name='action', match_param='action=edit', renderer='templates/edit.jinja2')
 def update(request):
-    return 'edit page'
+    string = request.query_string
+    __, req_id = string.split('=')
+    entry = Entry.by_id(req_id)
+    form = EntryCreateForm(request.POST, entry)
```

Comment:
What happens in this case if the value you got for the entry id is not valid?  When `Entry.by_id(id)` returns None?